### ISSUE TITLE
Are javadocs required for the project?

### ISSUE THREAD

**Issue by swiftguy1** [nus-cs2030/2122-s1#332]
# Summary
Put a summary of the question

## Description 
Describe the problem and code snippets if necessary 

## Screenshots (if any): 
Insert Images here if necessary

**Comment by lelerer** [comment]
My TA said no need.

**Comment by DankaiAndrew** [comment]
i did not include Javadoc in my project and codecrunch accepted it so I guess not

**Comment by seasonedlasagna** [comment]
I think it's only needed for public methods.

**Comment by lioweugene** [comment]
yeah only needed for public methods. Safer to just include Javadocs
you can refer to this!
[https://www.comp.nus.edu.sg/~cs2030/javadoc/]

**Comment by kxshxsh** [comment]
"I realise prof said yst he disabled javadocs for project
So nope dont need"
                                - my TA

**Comment by albertsutz** [comment]
Just do the public methods. You will be required to do those things anyway (For Checkstyle to pass). Other than that, you can get grade A even if you don't do javadocs for others) So It's okay.

**Comment by audrey-eng** [comment]
yup you will have to add in javadocs for public class, constructors and methods, can take a look at this link - https://nus-cs2030.github.io/1718-s2/javadoc/index.html

**Comment by kxshxsh** [comment]
> Just do the public methods. You will be required to do those things anyway (For Checkstyle to pass). Other than that, you can get grade A even if you don't do javadocs for others) So It's okay.

No need actually.  I packed checkstyle without using javadocs and got an A
